pkg/aws/client_test.go

```diff
@@ -38,6 +41,57 @@ var _ = Describe("Client", func() {
 		mockCtrl.Finish()
 	})
 
+	Context("ValidateCFUserCredentials", func() {
+		var (
+			CFsecretID      = "sut"
+			IAMAccessKey    = "sut"
+			oddIAMAccessKey = "longtestkey"
+		)
+		Context("when creds are OK and matches", func() {
+			BeforeEach(func() {
+				mockCfAPI.EXPECT().DescribeStackResource(gomock.Any()).Return(&cloudformation.DescribeStackResourceOutput{
+					StackResourceDetail: &cloudformation.StackResourceDetail{
+						PhysicalResourceId: &CFsecretID,
+					},
+				}, nil)
+
+				mockIamAPI.EXPECT().ListAccessKeys(gomock.Any()).Return(&iam.ListAccessKeysOutput{
+					AccessKeyMetadata: []*iam.AccessKeyMetadata{
+						{
+							AccessKeyId: &IAMAccessKey,
+						},
+					},
+				}, nil)
+			})
+			It("should return nil", func() {
```

Comment:
I would add that returning `nil` is expected behaviour:
```suggestion
			It("should finish successfully and return nil", func() {
```